src/main/joyride/life_cycle.cljs

```diff
@@ -0,0 +1,37 @@
+(ns joyride.life-cycle
+  (:require ["path" :as path]
+            ["vscode" :as vscode]
+            [joyride.config :as conf]
+            [joyride.utils :as utils]
+            [promesa.core :as p]))
+
+(def user-init-script "activate.cljs")
+(def user-init-script-path (path/join conf/user-config-path
+                                      conf/user-scripts-path
+                                      user-init-script))
+(def user-init-script-abs-path user-init-script-path)
```

Comment:
Why is this var the same as the other one?

Reply:
Yeah, I should just reference the other one twice in the config map. Thanks!